Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Andy Grover <agrover@redhat.com>", "Mulhern <amulhern@redhat.com>",
 dbus = "0.5.2"
 clap = "1"
 nix = "0"
-devicemapper = "0.7"
+devicemapper = { git = "https://github.com/mulkieran/devicemapper-rs", branch = "master-trait" }
```

Comment:
you'll need to update this before merge

Reply:
Yes. I expect we'll do a new release for the trait stuff, so that this will point at 0.8 release.